src/OmniSharp.Stdio/OmniSharp.Stdio.csproj

```diff
@@ -1,19 +1,22 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
-  <PropertyGroup>
-    <TargetFramework>net46</TargetFramework>
-    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
-    <PlatformTarget>AnyCPU</PlatformTarget>
-  </PropertyGroup>
+    <PropertyGroup>
+        <TargetFramework>net46</TargetFramework>
+        <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+        <PlatformTarget>AnyCPU</PlatformTarget>
+        <AssemblyName>OmniSharp</AssemblyName>
```

Comment:
It looks like you *are* changing this to `OmniSharp`. In that case, could we rename the project too?

Reply:
This was based on @filipw's comment at one point about using a different executable name.  So this was intenation. And also happens to OmniSharp.Http, though I'm fine with changing if we want.

Reply:
Well, it kind of defeats your argument about obscuring the fact that we have different hosts. :smile:

That said, I'm OK if we don't rename the project. I'm fine with it.